applications/multiai_ultrasound/CMakeLists.txt

```diff
@@ -36,8 +36,10 @@ if(HOLOHUB_DOWNLOAD_DATASETS)
   )
 endif()
 
-add_subdirectory(operators)
-add_subdirectory(cpp)
+if(NOT HOLOHUB_WHEEL_TESTING)
+    add_subdirectory(operators)
+    add_subdirectory(cpp)
+endif()
```

Comment:
Seems like this would break `multiai_ultrasound` in Python testing. `multiai_ultrasound` depends on Python-wrapped C++ operators. Based on that it doesn't seem like `multiai_ultrasound` is a valid candidate for Python wheel test coverage.